The function getdelim() behaves slightly different on FreeBSD,
only appending to the *line buffer if line_size is 0.

See: 
https://savannah.nongnu.org/bugs/?29466
http://bugs.gentoo.org/314791

Already comitted upstream:
http://cvs.savannah.gnu.org/viewvc/ccvs/src/myndbm.c?root=cvs&r1=1.38&r2=1.39

--- src/myndbm.c.orig
+++ src/myndbm.c
@@ -213,7 +213,7 @@
 mydbm_load_file (FILE *fp, List *list, char *filename)
 {
     char *line = NULL;
-    size_t line_size;
+    size_t line_size = 0;
     char *value;
     size_t value_allocated;
     char *cp, *vp;
